### PR TITLE
Fix for a small cube not having visibility points.

### PR DESCRIPTION
**Pull request by rartiss55**
Also fixes null pointer exception in case there are no visibility points set.

### CODE REVIEW

File: unity/Assets/Scripts/MCSMain.cs [hunk 1]
```diff
@@ -1000,6 +1000,17 @@ MCSConfigObjectDefinition objectDefinition
         return point;
     }
 
+    private float calculateCubeGridValue(float size) {
+        float cubeInternalGrid = MCSMain.CUBE_INTERNAL_GRID;
+
+        // Ensure visibility points still set for very small cubes
+        if (size == MCSMain.CUBE_INTERNAL_GRID) {
```
Comment (ThomasSchellenbergNextCentury) [inline unity/Assets/Scripts/MCSMain.cs:1007]: Shouldn't this be something like `<=`?

Comment (rartiss55) [reply unity/Assets/Scripts/MCSMain.cs:1007]: I felt like no but I could be wrong. The particular issue here was that x/y/z size of .25 / cube internal grid of .25 = 1, which was too large a number for the x/y/z values in the for loops below that generate the visibility points (looking for x/y/z < 1). So I thought the only issue would be when the size is exactly equal to the grid. Other values less than CUBE_INTERNAL_GRID would probably be okay, although the visibility point placement probably wouldn't be exactly in the middle.  

If we made that check <=, you could run into the same issue with a smaller number (like with a size value of 0.125).

Comment (rartiss55) [reply unity/Assets/Scripts/MCSMain.cs:1007]: Taking a closer look now...

Comment (rartiss55) [reply unity/Assets/Scripts/MCSMain.cs]: Okay, now it should be fixed.

**Pull request merged**